src/Sulu/Bundle/MediaBundle/Resources/public/scss/modules/area-selection.scss

```diff
@@ -0,0 +1,134 @@
+.sulu-area-selection {
+    padding: 10px;
+    width: 100%;
+    height: 100%;
+    text-align: center;
+    white-space: nowrap;
+    @include user-select(none);
+
+    &::before {
+        content: '';
+        display: inline-block;
+        height: 100%;
+        vertical-align: middle;
+        margin-right: -0.25em;
+    }
+
+    .frame {
+        position: relative;
+        display: inline-block;
+        vertical-align: middle;
+    }
+
+    .area {
+        position: absolute;
+        top: 0; // gets overwritten in the components javascript
+        left: 0; // gets overwritten in the components javascript
+        width: 0; // gets overwritten in the components javascript
+        height: 0; // gets overwritten in the components javascript
```

Comment:
Couldn't you place all these comments in a single line before these values, instead of repeating it?